book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -19,13 +19,14 @@ You'll do the following:
 ==== Basic Branching
 
 (((branches, basic workflow)))
-First, let's say you're working on your project and have a couple of commits already.
+First, let's say you're working on your project and have a couple of commits already
+on the `master` branch.
 
 .A simple commit history
 image::images/basic-branching-1.png[A simple commit history.]
 
 You've decided that you're going to work on issue #53 in whatever issue-tracking system your company uses.
-To create a branch and switch to it at the same time, you can run the `git checkout` command with the `-b` switch:
+To create a new branch (relative to your current location) and switch to it at the same time, you can run the `git checkout` command with the `-b` switch:
```

Comment:
Let's leave out the parenthetical here. It muddies the water by introducing the concept of "location," which isn't really a core concept in Git. It will become clear by example and repetition that `git branch` and `git checkout -b` will create branches at the commit pointed to by `HEAD`.